css/comp_lib_styles/components/grid.css

```diff
@@ -0,0 +1,11 @@
+.grid-col-2 {
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: 1fr 1fr;
```

Comment:
You can use `grid-template-columns: repeat(2, 1fr)` here. No difference really, just makes it a bit more clean and crisp.